pyteal/ast/router.py

```diff
@@ -133,7 +133,7 @@ class OnCompleteAction:
     def __post_init__(self):
         if bool(self.call_config) ^ bool(self.action):
             raise TealInputError(
-                f"action {self.action} and call_config {str(self.call_config)} contradicts"
+                f"action {self.action} and call_config {self.call_config!r} contradicts"
```

Comment:
are you intentionally preferring `repr(self.call_config)` over `str(self.call_config)`? If so, this is great. If you just want `str()`, then you can just do `{self.call_config}` and not call `str()` on it.